instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -36,12 +36,13 @@ def parse(query_string:, &block)
           def validate(query:, validate:, &block)
             tracer.in_span('graphql.validate') do |span|
               super.tap do |response|
-                errors = response[:errors]&.compact&.map(&:to_h)&.to_json
-                unless errors.nil?
+                errors = response[:errors]&.compact&.map(&:to_h) || []
+
+                unless errors.empty?
                   span.add_event(
                     'graphql.validation.error',
                     attributes: {
-                      'message' => errors
+                      'message' => errors.to_json
```

Comment:
What are your thoughts on using `exception.message` as a key here instead of `message`? This would more closely mirror the `record_exception` SDK method: https://github.com/open-telemetry/opentelemetry-ruby/blob/main/sdk/lib/opentelemetry/sdk/trace/span.rb#L164-L172

Reply:
💡 That is in fact what the JS implementation does https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts#L393

https://github.com/open-telemetry/opentelemetry-js/blob/2b20565d105d4267de5e19a730c2401eacad43a4/packages/opentelemetry-sdk-trace-base/src/Span.ts#L275

Good catch!

Reply:
```suggestion
                      'exception.message' => errors.to_json
```